tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator.Console/Program.cs

```diff
@@ -13,17 +13,25 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            if (args.Length < 2)
+            try
             {
-                System.Console.WriteLine("Usage: ");
-                System.Console.WriteLine("metadatagen <sourcepath> <output>");
+                if (args.Length < 2)
+                {
+                    System.Console.WriteLine("Usage: ");
+                    System.Console.WriteLine("metadatagen <sourcepath> <output>");
 
-                return;
-            }
+                    return;
+                }
 
-            string sourcePath = args[0];
-            AssemblyLoader.Initialize(sourcePath, Log);
-            ExtensionsMetadataGenerator.Generate(sourcePath, args[1], Log);
+                string sourcePath = args[0];
+                AssemblyLoader.Initialize(sourcePath, Log);
+                ExtensionsMetadataGenerator.Generate(sourcePath, args[1], Log);
+            }
+            catch (Exception ex)
+            {
+                Log("Error generating extension metadata: " + ex.ToString());
```

Comment:
Where would the logs appear, in VS debugging window?

Reply:
They get logged via MSBuild. So it'll show up in the VS Output window, or in the Errors/Warnings/Message pane, or in build output when building from command line.